src/utils/index.ts

```diff
@@ -289,33 +289,16 @@ export const isFormMessage = (message: CoreMessageType): boolean => !!(
 );
 
 export const isTemplateMessage = (message: CoreMessageType): boolean => !!(
+  // FIXME: change to message_template once server changes
   message && message.extendedMessagePayload?.['template']
```

Comment:
Currently, customer will send directly, so it seems fine to change it right away!
Also, it would be good to move the key to the constants file

Reply:
ah you are right. I will apply this, thanks!